build.gradle

```diff
@@ -52,7 +52,7 @@ artifacts {
 dependencies {
     // iexec
     compile "com.iexec.common:iexec-common:$iexecCommonVersion"
-    //compile files('../iexec-common/build/libs/iexec-common-3.0.0-SNAPSHOT.jar')
+    //compile files('../iexec-common/build/libs/iexec-common-${iexecCommonVersion}.jar')
```

Comment:
This wont compile, could you change your simple quotes to double quotes please?
(I can't find the `update code directly` Github feature we've been using)

Reply:
Nice catch, thanks. Fixed